### PR TITLE
Added restaurants and activities to htx file

### PR DESCRIPTION
**Pull request by zazazack**
Added restaurants and activities to htx file :watermelon: 

@githubteacher 

What do you think? :thought_balloon: 

Anything else to add?

### CODE REVIEW

File: zazazack-houston.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+### Restaurants
+Soooooo many restaurants in Houston.
+
+- Pink's Pizza
+- Jinya
+- El Tiempo
+- Beaver's BBQ
+- Hugo's
+- Cafe Brasil
+- ...
```
Comment (githubteacher) [inline zazazack-houston.md:12]: @zazazack Sounds like there are lots of awesome places to eat! 😄  👍

File: zazazack-houston.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+### Activities
+- Texans Game
+- Museum of Fine Arts
+- Houston Zoo
```
Comment (ghost) [inline zazazack-houston.md:19]: Which is your favourite place to eat?
What is at the Texans Game?

**Review by githubteacher — APPROVED** [review]
👍 Looks good to merge! 🚢 :shipit:

**Review by ghost — APPROVED** [review]
👍 Looks good to merge!

**Pull request merged**